mantidproject/mantid
# Add KAREN peak filter to DeltaPDF3D

See docs for complete description of KAREN method.

**To test:**
This is a large data set to try, it may take about 30 minutes.
```python
ZrO2=LoadMD(Filename='/SNS/users/rwp/corelli/ZrO2/IPTS-12310/ZrO2_300K_sym.nxs')
DeltaPDF3D(InputWorkspace='ZrO2', IntermediateWorkspace='intermediate', OutputWorkspace='output')
```

Compare `ZrO2` and `intermediate` and see that the Bragg peaks are mostly removed. Check the output looks about right.

Also try with `WindowFunction='None'`

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
